index.css

```diff
@@ -22,3 +22,12 @@ div {
   max-width: 800px;
   height: 500px;
 }
+
+.history {
+  margin-bottom: 1.5%;
+  margin-top: 2%;
+  color: #1a1b1c;
+  font-style: normal;
+  flex: none;
+  flex-grow: 0;
```

Comment:
### Add front styles

Reply:
Good job using inline comments :clap: ,however, I suggest doing the following to make it even better:
 - Your comment has to sound like a suggestion not like an order, instead of "Add front styles" You can say for example : "I think it would be better...", "I suggest adding..." and so on.
 - PLease rememer to add the reason why you suggest each change as it is an opportunity for the student to learn :thinking: